myapp/config/application.rb

```diff
@@ -14,7 +14,7 @@ class Application < Rails::Application
     # 言語を日本語に設定
     config.i18n.default_locale = :ja
     # localeファイルの読み込み設定
-    config.i18n.load_path += Dir[Rails.root.join('path/to')]
+    config.i18n.load_path += Dir[Rails.root.join('path/to', '**').to_s]
```

Comment:
i18nの対象箇所は下記の箇所になるはずですがどうでしょうか？
path/toがどこにあたるのか分からないので適切な箇所を指定してもらったほうがいいかと！
```suggestion
    config.i18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{yml}').to_s]
```

Reply:
適切なパスに修正しました。

Reply:
修正対応ありがとうございます！